### ISSUE TITLE
ppc64le: fail to boot with kernel 6.6.23, works with 6.6.18.

### ISSUE THREAD

**Issue by AlbertoGP** [chimera-linux/cports#1728]
I ran `apk upgrade` yesterday. This is the first boot after that, now with kernel `6.6.23`.
Disks are all encrypted with LUKS, booting process stops at “waiting for encrypted disk [...]”.
The file system is XFS.

It booted after selecting the previous kernel version (`6.6.18`) in recovery mode, everything seems to be working as usual. I do not know whether plain 6.6.18 works but I will try it next time I start the machine, I have no reason to think it would not.

I guess it is some issue with XFS in the new kernel, such things have happened on occasion with void-ppc.

If there are some diagnostics I could perform, please let me know.
This is not urgent for me as long as it works by selecting 6.6.18 in the boot menu.

**Comment by q66** [comment]
can you try linux-stable?

**Comment by AlbertoGP** [comment]
I thought this meant that it was installed:
```
> apk info linux-stable
linux-stable-6.8.2-r0 description:
Linux kernel 6.8.x

linux-stable-6.8.2-r0 webpage:
https://kernel.org

linux-stable-6.8.2-r0 installed size:
131 MiB
```
… but I just did `apk add linux-stable` and it did install. The `config`, `initrd`, and `vmlinux` files for `6.8.2` are now in `/boot/`, they were not before.

I’m going to reboot now and see how it goes.

**Comment by q66** [comment]
you should also try if it's really the kernel and not some change in the initramfs; first backup your existing initramfs (so you have recovery if needed; just copy it to another file in the same dir, you can change the petitboot boot option in case of failure after regen) and then run `update-initramfs -c -k 6.6.18-0-generic` and try booting again

**Comment by AlbertoGP** [comment]
Ok, I’m going to do that too before rebooting.

**Comment by q66** [comment]
i meant backing up/updating initramfs for 6.6.18 to ensure that something non-kernel did not break, but a bad crypttab could also be it

**Comment by AlbertoGP** [comment]
Ah, yes at the end I thought that I should follow your instructions exactly so I did back up both `6.6.18` and `6.6.23`.

After running `update-initramfs` for `6.6.23` and `6.8.2` it boots perfectly with both: `6.6.23` selecting it in the boot menu, and `6.8.2` by default. Therefore I’m closing this issue report.

It was the `crypttab` entry using `/dev/sda2` which had changed to `/dev/sdb2` after I added a new data SSD. The boot disk is the same and stayed plugged in the same SATA port, it just got a new device name after the other got installed.

For the other disks I use `UUID=…` but this entry was the one made by the installer and I failed to notice it until today.

Additional good news: now the AMD RX 5700 card works without having to disable the built-in VGA.
I don’t know whether this is something you did in Chimera, or the new kernel, or something else, but it is a great relief.

Thank you for all your hard work and your patience.
